dev/tasks/conda-recipes/arrow-cpp/bld.bat

```diff
@@ -18,6 +18,7 @@ cmake -G "Ninja" ^
       -DARROW_BUILD_UTILITIES:BOOL=OFF ^
       -DARROW_BUILD_STATIC:BOOL=OFF ^
       -DCMAKE_BUILD_TYPE=release ^
+      -DARROW_SSE42:BOOL=OFF ^
```

Comment:
Can you rebase and change this to be `ARROW_SIMD_LEVEL=NONE`?

Reply:
Done. (And sorted.)